packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ImageAndTextViewer/components/ImageAndTextControls.js

```diff
@@ -0,0 +1,237 @@
+import { Button, Box, ThemeContext } from 'grommet'
+import {
+  DocumentText,
+  FormNext,
+  FormPrevious
+} from 'grommet-icons'
+import PropTypes from 'prop-types'
+import { useEffect, useRef, useState } from 'react';
+import styled, { withTheme, css } from 'styled-components'
+import { useTranslation } from '@translations/i18n'
+
+import controlsTheme from './theme'
+import locationValidator from '../../../helpers/locationValidator'
+
+const DirectionButton = styled(Button)`
+  border: none;
+  padding: 0;
+    & > div {
+      flex-direction: column;
+    }
+`
+
+const ThumbnailButton = styled(Button)`
+  display: flex;
+  ${props => css`height: ${props.thumbnailDimension};`}
+  ${props => css`width: ${props.thumbnailDimension};`}
+  padding: 0;
+  border: none;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  ${props => css`background-image: url(${props.thumbnailerUrl});`}
+  &[aria-selected=true] {
+    border: solid 2px ${props => props.theme.global.colors['neutral-2']};
+  }
+`
+
+const TextButton = styled(Button)`
+  align-items: center;
+  border: solid 1px ${props => props.theme.global.colors['light-3']};
+  display: flex;
+  justify-content: center;
+  padding: 0;
+  ${props => css`height: ${props.thumbnailDimension};`}
+  ${props => css`width: ${props.thumbnailDimension};`}
+  &:hover {
+    box-shadow: 0px 0px 0px 2px ${props => props.theme.global.colors['brand']};
+  }
+  &[aria-selected=true] {
+    border: solid 2px ${props => props.theme.global.colors['neutral-2']};
+  }
+`
+
+const backgrounds = { dark: 'dark-3', light: 'neutral-6' }
+
+const ImageAndTextControls = ({
+  currentFrame = 0,
+  locations = [],
+  onFrameChange = () => true
+}) => {
+  const { t } = useTranslation('components')
+  
+  /** DefaultLayout for classify page styling has breakpoints at 700px and 1160px
+   * In ImageAndTextControls we're simply checking for when this component is < 450px
+   * which happens both before and after the 700px layout breakpoint
+   */
```

Comment:
```suggestion
  /** DefaultLayout for classify page styling has breakpoints at 768px and 1160px
   * In ImageAndTextControls we're simply checking for when this component is < 450px
   * which happens both before and after the 768px layout breakpoint
   */
```

This note will change with the merging of PR #4066. Probably should update the code comment in FlipbookControls as well if this comment was copy pasted from there.